app/src/main/res/values-pt/strings.xml

```diff
@@ -1,272 +1,543 @@
-    <string name="login">Login</string>
-    <string name="toast_welcome">Welcome %1$s</string>
-    <string name="unable_to_connect">Not connected to internet</string>
-    <string name="basic">Basic</string>
-    <string name="username">Username</string>
-    <string name="progress_message_login">Logging In</string>
-    <string name="progress_message_loading">Loading</string>
-    <string name="password">Password</string>
+    <string name="app_name">Mifos Mobile</string>
+    <string name="login">Entrar</string>
+    <string name="toast_welcome">Bem-vindo %1$s</string>
+    <string name="unable_to_connect">Não há ligação à internet</string>
+    <string name="basic">Básico</string>
+    <string name="username">Nome de Utilizador</string>
+    <string name="progress_message_login">A entrar</string>
+    <string name="progress_message_loading">A carregar</string>
+    <string name="password">Palavra-passe</string>
     <string name="email">Email</string>
-    <string name="accounts">Accounts</string>
-    <string name="clients">Clients</string>
-    <string name="funds_transfer">Funds Transfer</string>
-    <string name="recent_transactions">Recent Transactions</string>
-    <string name="charges">Charges</string>
-    <string name="questionnaire">Questionnaire</string>
-    <string name="about_us">About Us</string>
-    <string name="saving_account_details">Saving Account Details</string>
-    <string name="savings_account">Savings Account</string>
-    <string name="error_loan_account_details_loading">Error loading in loan accounts detail</string>
-    <string name="nominal_interest_rate">Nominal Interest Rate</string>
-    <string name="account_number">Account Number </string>
-    <string name="account_balance">Account Balance</string>
-    <string name="total_deposits">Total Deposits</string>
-    <string name="open_drawer">Open Drawer</string>
-    <string name="close_drawer">Close Drawer</string>
+    <string name="accounts">Contas</string>
+    <string name="clients">Clientes</string>
+    <string name="funds_transfer">Transferência de Fundos</string>
+    <string name="recent_transactions">Transações recentes</string>
+    <string name="charges">Cobranças</string>
+    <string name="questionnaire">Questionário</string>
+    <string name="about_us">Sobre nós</string>
+    <string name="saving_account_details">Detalhes da Conta Poupança</string>
+    <string name="savings_account">Conta Poupança</string>
+    <string name="error_loan_account_details_loading">Erro ao carregar detalhes da Conta de Empréstimo</string>
+    <string name="nominal_interest_rate">Taxa de Interesse Nominal</string>
+    <string name="account_number">Conta número </string>
+    <string name="account_balance">Saldo da Conta</string>
+    <string name="total_deposits">Depósito Total</string>
+    <string name="open_drawer">Abrir Caixa</string>
+    <string name="close_drawer">Fechar Caixa</string>
     <string name="home">Home</string>
```

Comment:
Why you haven't translated this string and many others like this too, any specific reason for that?

Reply:
Yes. They do not have a literal translation and therefore cannot be translated. Example: "Home" translated into portuguese means only "own house" and there is no other word to translate with the same meaning. Besides, "home" is kind of a world loan from English. Most of Portuguese people understand what it is and what it means. We use a lot of English words in our quotidian.

"Splash" for example would be suicide to translate. It can be translated into its literal but nobody would associate it with the real meaning. Most certainly everyone would associate it with this:

![image](https://user-images.githubusercontent.com/6632800/48980082-74450b80-f0bc-11e8-8151-8c2e73ad648f.png)

So, the best option is to keep it untranslated. Like I have said before, in Portugal we use a lot of English words in our everyday. There is no problem.